php/class-terminus-command.php

```diff
@@ -1,34 +1,27 @@
 <?php
 use \Terminus\Endpoint;
 use \Terminus\Request;
-use \Terminus\Fixtures;
 use \Terminus\Session;
 use \Terminus\Auth;
 use \Terminus\Loggers\Regular as Logger;
-
```

Comment:
There need to be blank lines between functions and between functions and instance variables.